docs/user-guides/b-kepler-gl-workflow/b-add-data-layers/a-adding-data-layers.md

```diff
@@ -8,6 +8,6 @@ Each blue dot represents the point (latitude and longitude). Layers work like pa
 
 ![Sample NYC Map with colors](https://d1a3f4spazzrp4.cloudfront.net/kepler.gl/documentation/image6.png "Sample NYC Map with colors")
 
-Learn more about the [types of layers] available in Kepler.gl.
+Learn more about the [types of layers](../../c-types-of-layers.md) available in Kepler.gl.
```

Comment:
I am not seeing this link in the directory structure. I do see a folder but it's not not gonna work